collections/_pages/talks-and-presentations.html

```diff
@@ -102,34 +101,33 @@
 
 <section class="event">
   <div class="container flex">
-    <div class="text ">
+    <div class="text " style="text-align: justify; text-justify: inter-word">
       <h2>CNCF On-Demand Webinar: <strong>Meshery - The Service Mesh Manager</strong></h2>
       <p> This webinar introduces Meshery, an open-source, multi-service mesh management plane that can provision nine different service meshes, their sample applications and benchmarks the performance of service mesh deployments.
         <br>
-      <div align="center" class="button-para" style="margin-top: 3rem;">
-        <a href="#open-modal" class="link">Find video and slides here</a>  
-      </div>
+        <div class="button-para" style="margin-top: 3rem; ">
+          <button class="link" style="position: relative;left: 25%;">Find video and slides here</button>  
```

Comment:
![mobile2](https://user-images.githubusercontent.com/85789734/140069553-c37895be-cb3b-4b06-b7fb-880d6a31f904.jpg)
The button should be aligned to the center in mobile view too.  Also, can you remove the inline styles and add them inside the style tag.